src/pages/dar_application/ResearcherInfo.jsx

```diff
@@ -0,0 +1,394 @@
+            </div>
+          )}
+          <CollaboratorList
+            formFieldChange={formFieldChange}
+            collaborators={formData.internalCollaborators}
+            collaboratorKey='internalCollaborators'
+            collaboratorLabel='Internal Collaborator'
+            setCompleted={setInternalCollaboratorsCompleted}
+            validation={validation.internalCollaborators}
+            onValidationChange={onValidationChange}
+            showApproval={false}
+            disabled={!isEmpty(darCode) || readOnlyMode}
+          />
+        </div>
+
+        <div className='dar-application-row'>
+          <FormField
+            id='signingOfficial'
+            type={FormFieldTypes.SELECT}
+            description='I certify that the individual listed below is my Institutional Signing official'
+            title='1.6 Institutional Signing Official'
+            validators={[FormValidators.REQUIRED]}
+            ariaLevel={ariaLevel + 1}
+            defaultValue={formData.signingOfficial}
+            validation={validation.signingOfficial}
+            onValidationChange={onValidationChange}
+            disabled={readOnlyMode}
+            onChange={({ key, value }) => {
+              formFieldChange({ key, value });
+            }}
+            selectOptions={(allSigningOfficials?.map((so) => {
+              return formatSOString(so.displayName, so.email);
+            }) || [''])}
+          />
+        </div>
+
+        <div className='dar-application-row'>
+          <FormField
+            id='itDirector'
+            disabled={readOnlyMode}
+            description='I certify that the individual listed below is my IT Director'
+            placeholder='Enter Firstname Lastname'
+            title='1.7 Information Technology (IT) Director'
+            validators={[FormValidators.REQUIRED]}
+            ariaLevel={ariaLevel + 1}
+            validation={validation.itDirector}
+            onValidationChange={onValidationChange}
+            onChange={({ key, value }) => formFieldChange({ key, value })}
+            defaultValue={formData.itDirector}
+          />
+        </div>
+
+        <div className='dar-application-row'>
+          <div>
+            <FormField
+              id='anvilUse'
+              disabled={readOnlyMode}
+              type={FormFieldTypes.RADIOGROUP}
+              title='1.8 Cloud Use Statement'
+              description={[
+                <span key='anvil-use-description'>
+                  {'Will you perform all of your data storage and analysis for this project on the '}
+                  <a rel='noopener noreferrer' href='https://anvil.terra.bio/' target='_blank'>AnVIL</a>
+                  {'?'}
```

Comment:
See previous feedback - I think this is better expressed without the blocks and using &nbsp; before the AnVIL link.